lib/api/v3/work_packages/work_package_contract.rb

```diff
@@ -27,28 +27,39 @@
 # See doc/COPYRIGHT.rdoc for more details.
 #++
 
-require 'reform'
-require 'reform/form/active_model/model_validations'
-
 module API
   module V3
     module WorkPackages
-      class WorkPackageContract < Reform::Contract
-        def self.writable_attributes
-          @writable_attributes ||= %w(
-            lock_version
-            subject
-            parent_id
-            description
-            start_date
-            due_date
-            status_id
-            assigned_to_id
-            responsible_id
-            priority_id
-            category_id
-            fixed_version_id
-          )
+      class WorkPackageContract < ::API::Contracts::ModelContract
+        writable_attribute :subject
```

Comment:
Sounds good to me,

however I'd rather not use that feature too extensively, because I feel that the readability does not increase very much, when you put a bunch of unrelated attributes into one line.